public/app/features/users/UsersTable.test.tsx

```diff
@@ -15,39 +15,45 @@ jest.mock('app/core/core', () => ({
   },
 }));
 
-const setup = (propOverrides?: object) => {
+function getProps(propOverrides?: object) {
   const props: Props = {
     users: [] as OrgUser[],
     onRoleChange: jest.fn(),
     onRemoveUser: jest.fn(),
   };
 
-  Object.assign(props, propOverrides);
-
-  return shallow(<UsersTable {...props} />);
-};
+  return Object.assign(props, propOverrides);
+}
 
 describe('Render', () => {
   it('should render component', () => {
-    const wrapper = setup();
-
-    expect(wrapper).toMatchSnapshot();
+    render(<UsersTable {...getProps()} />);
   });
 
   it('should render users table', () => {
-    const wrapper = setup({
-      users: getMockUsers(5),
-    });
-
-    expect(wrapper).toMatchSnapshot();
+    render(
```

Comment:
same here, no `expect`.

sounds like it wants to check that the users are rendered in the table, so maybe something like:

```typescript
const users = getMockUsers(5);
render(<UsersTable {...getProps({ users })} />);
users.forEach((user) => {
  expect(screen.getByText(user.name)).toBeInTheDocument();
])
```

Reply:
Yes! Okay. I wasn't sure how much should I rely on the current implementation, e.g. the fact that each user's name is visible only ones in the DOM right now